server/src/async/index.js

```diff
@@ -0,0 +1,22 @@
+const cron = require('node-schedule');
+
+const emitter = require('./emitter');
+const events = require('./events');
+
+function runJobs() {
+  console.log('Jobs are in a queue.');
+
+  const EVERY_DAY_AT_MIDNIGHT = '0 0 * * *';
```

Comment:
Can we run this every hour instead? Also, is it possible to trigger remotely for testing?